src/test/java/nextstep/ladder/NameTest.java

```diff
@@ -0,0 +1,39 @@
+package nextstep.ladder;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+
+import static org.assertj.core.api.Assertions.*;
+
+@DisplayName("Name 클래스 테스트")
+public class NameTest {
+
+    @Test
+    @DisplayName("Name_생성_테스트")
+    public void Name_생성_테스트() {
+        Name name = new Name("test");
+        assertThat(name.toString()).isEqualTo("test");
+    }
+
+
+    @ParameterizedTest(name = "[index] name = {0}")
+    @NullAndEmptySource
+    @DisplayName("Name_Null_Black_테스트")
+    public void Name_Null_Black_테스트(String name) {
+        assertThatIllegalArgumentException()
+                .isThrownBy(() -> {
+                    Name person = new Name(name);
```

Comment:
생성자 테스트이니 로컬변수와 할당 코드는 제거하면 어떨까요?
```suggestion
                    new Name(name);
```